cockroachcloud/configure-cloud-org-sso.md

```diff
@@ -0,0 +1,213 @@
+
+## Plan to enable Cloud Organization SSO
+
+To ensure a smooth migration to Cloud Organization SSO, review the following information before you enable the feature.
+
+{{site.data.alerts.callout_info}}
+After it is enabled, Cloud Organization SSO **cannot** be disabled. To configure Cloud Organization SSO to behave like [Basic SSO](cloud-org-sso.html#basic-sso), refer to [Emulate Basic SSO](#emulate-basic-sso).
+{{site.data.alerts.end}}
+
+### Decide your custom URL
+
+When you [enable Cloud Organization SSO](#enable-cloud-organization-sso), you will configure a custom URL through which your users can access {{ site.data.products.db }}. After the feature is enabled, your users must use that page to sign in. Your custom URL must be unique within {{ site.data.products.db }}. **Communicate this custom URL to your users ahead of time.**
+
+To change your custom URL after enabling Cloud Organization SSO, contact your account team.
+
+### Decide which authentication methods to enable
+
+All enabled authentication methods appear on your custom URL and are available to your users. By default, when you enable Cloud Organization SSO, the following authentication methods are enabled by default:
+
+- Password
+- GitHub
+- Google
+- Microsoft
+
+In addition, you can create authentication methods that connect to your identity provider using the [Security Access Markup Language (SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) and [OpenID Connect (OIDC)](https://openid.net/connect/) identity protocols.
+
+Members are identified by their email address. To allow members to migrate from password authentication to SSO, ensure that their email addresses in your {{ site.data.products.db }} organization match those in your identity providers. To allow your members to select from multiple SSO authentication methods, ensure that the email addresses match across all of them.
+
+### Communicate to your members
+
+Before you enable Cloud Organization SSO, notify your users about what to expect, such as:
+
+- The custom login URL and when they should begin using it.
+- Which authentication methods they can use and whether they have autoprovisioning enabled.
+- Some members may need to be re-added to your organization:
+  - All members of your {{ site.data.products.db }} organization who were using [Basic SSO](cloud-org-sso.html#basic-sso) rather than an email and password must sign in again to regain access to your organization. After signing in, members retain the same access they had before the migration.
+  - Members who are also members of other organizations must be re-added to your organization. If they sign in using an authentication method with [autoprovisioning](#autoprovisioning) enabled, they are automatically added upon successful sign-in. Otherwise, they must be re-invited. If they previously had the organizational admin role, it must be granted to them again.
```

Comment:
Is "organizational admin role" the same thing as what we usually call `admin` role, or something else? If the former, then I think it needs the backticks here & elsewhere.